src/browser/CMakeLists.txt

```diff
@@ -29,8 +28,19 @@ if(WITH_XC_BROWSER)
             BrowserService.cpp
             BrowserSettings.cpp
             BrowserShared.cpp
-            NativeMessageInstaller.cpp
-            )
+            NativeMessageInstaller.cpp)
+
+    if(WITH_XC_BROWSER_PASSKEYS)
+        # CBOR requires Qt 5.12
+        if(Qt5Core_VERSION VERSION_LESS "5.12.0")
+            message(FATAL_ERROR "Qt version 5.12.0 or higher is required for Passkeys support")
+        endif()
```

Comment:
This should be moved to the root CMakeLists.txt as an initial check to see if we can even use WITH_XC_BROWSER_PASSKEYS. If the check fails, just issue an _info message_ and set WITH_XC_BROWSER_PASSKEYS to OFF.

Reply:
I'll remove it when we make that transition